docs/guides/opera_rtc_product_guide.md

```diff
@@ -219,7 +218,7 @@ Products generated On Demand using ASF's HyP3 platform are delivered as zip file
 available for the products generated by the OPERA SDS. The zip file also includes the color browse image (which is 
 not georeferenced) used for displaying archived OPERA RTC-S1 products in Vertex. 
 
-The naming convention for OPERA RTC-S1 products generated On-Demand using ASF's HyP3 platform is the same as for 
+The naming convention for OPERA RTC-S1 products generated On Demand using ASF's HyP3 platform is the same as for 
```

Comment:
should this be lower-case, since it's not referring to the specific product? 

```suggestion
The naming convention for OPERA RTC-S1 products generated on demand using ASF's HyP3 platform is the same as for 
```

Reply:
The branding 'On Demand' is problematic on a number of grammatical/convention fronts. I tend towards just capitalizing whenever we're talking about HyP3-generated products. I looked through this document, and there was only one place where it wasn't capitalized. I've changed that, so at least it's consistent within this document.